src/static/js/modules/init-chosen.js

```diff
@@ -10,7 +10,9 @@ require( 'chosen' );
 function init() {
   $( '.chosen-select' ).chosen( {
     width:           '100%',
+    /* eslint-disable */
```

Comment:
So not awesome, but you can do either of the following which may provide some indication of why the ignore is there:

```
function init() {
  $( '.chosen-select' ).chosen( {
    width:           '100%',
    no_results_text: 'Oops, nothing found!' // eslint-disable-line camelcase, no-inline-comments, max-len
  } );
}
```

…or…

```
function init() {
  $( '.chosen-select' ).chosen( {
    width:           '100%',
    /* eslint-disable camelcase, lines-around-comment */
    no_results_text: 'Oops, nothing found!'
    /* eslint-endable */
  } );
}
```